ChatTTS/utils/gpu.py

```diff
@@ -34,7 +44,34 @@ def select_device(min_memory=2047, experimental=False):
             logger.get_logger().info("found Apple GPU, but use CPU.")
             device = torch.device("cpu")
     else:
-        logger.get_logger().warning("no GPU found, use CPU instead")
+        logger.get_logger().warning("no GPU or NPU found, use CPU instead")
         device = torch.device("cpu")
 
     return device
+
+
+def _is_torch_npu_available():
+    """
+    check if torch_npu is installed and if a NPU is in the environment
+    """
+    try:
```

Comment:
You can put this try out of the function definition. Instead, put it at the top of this file, in case user defining npu manually.